src/App.js

```diff
@@ -837,7 +847,7 @@ const HeroBodyPart = React.forwardRef(
     pos.add(offset);
     const [, api] = useBox(
       () => ({
-        mass: shape.mass || shape.size[0] * shape.size[1] * shape.size[2],
+        mass: 5 * (shape.mass || shape.size[0] * shape.size[1] * shape.size[2]),
```

Comment:
Wow. 👍  Somehow reminds me of https://profmattstrassler.com/articles-and-posts/particle-physics-basics/the-hierarchy-problem/naturalness/.